src/components/pages/dashboard-pages/EditLesson.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState, useEffect } from 'react';
+import { useNavigate, useLocation } from 'react-router-dom';
+import { getLessonDetails, editLesson } from '../../../util/DataBaseRequests';
+import { useAuth } from '../../../AuthProvider';
+import EditLessonForm from './EditLessonForm';
+
+const EditLesson = () => {
+  const navigate = useNavigate();
+  const { userData } = useAuth();
+  const location = useLocation();
+  const { selectedStudentId, lessonId } = location.state || {};
+
+  const [formData, setFormData] = useState({
+    lessonTitle: '',
+    classId: '',
+    lessonDescription: '',
+    type: '',
+    hometask: '',
+    lessonSchedule: { date: '', startTime: '' },
+  });
+  const [formErrors, setFormErrors] = useState({});
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    const fetchClasses = async () => {
```

Comment:
The name is a little misleading b/c we are actually fetching lesson details and returning lesson info.

Reply:
Fixed